core-services/prow/02_config/openshift/openshift-controller-manager/_pluginconfig.yaml

```diff
@@ -13,6 +11,13 @@ label:
       - s-urbaniak
       - tkashem
       label: backport-risk-assessed
+    - allowed_users:
+      - jitendar-singh
+      - prietyc123
+      - xingxingxia
+      - wangke19
+      - gangwgr
```

Comment:
/cc @xingxingxia @wangke19 @gangwgr 

Adding you as cherry pick approvers to `openshift-controller-manager`, as your team Z-stream approvers are also on this repo.

Reply:
@adambkaplan OK, no problem for us.

Reply:
@adambkaplan OK, no problem for us.
/lgtm

Reply:
Agree with @xingxingxia